.github/workflows/linux-check.yaml

```diff
@@ -163,4 +163,5 @@ jobs:
           # world_feed_integration_tests - https://github.com/organicmaps/organicmaps/issues/215
           CTEST_EXCLUDE_REGEX: "drape_tests|generator_integration_tests|opening_hours_integration_tests|opening_hours_supported_features_tests|routing_benchmarks|routing_integration_tests|routing_quality_tests|search_quality_tests|storage_integration_tests|shaders_tests|world_feed_integration_tests"
         run: |
+          ln -s ../data data
```

Comment:
What's the reason in these lines?
Data and resource paths are directly passed to the test.
```cmake
if (NOT boost_test)
    set(test_command ${name} --data_path=${OMIM_DATA_DIR} --user_resource_path=${OMIM_USER_RESOURCES_DIR})
  else()
    set(test_command ${name})
  endif()
```

Reply:
The test does not use paths passed from the command line. @vng can we reuse these paths?

Reply:
- simplicity
- current fancy toggle NO_PLATFORM_INIT. data_path and user_resource_path are useless without Platform